### ISSUE TITLE
Use coverlet.collector built to master failed

### ISSUE THREAD

**Issue by Corniel** [Qowaiv/Qowaiv#414]
```
Run coverallsapp/github-action@master
  with:
    github-token: ***
    path-to-lcov: ./specs/Qowaiv.Specs/**/coverage.info
    coveralls-endpoint: https://coveralls.io/
    debug: false
  env:
    DOTNET_ROOT: /home/runner/.dotnet
Using lcov file: ./specs/Qowaiv.Specs/**/coverage.info
Error: ENOENT: no such file or directory, open './specs/Qowaiv.Specs/**/coverage.info'
```
See: https://github.com/Qowaiv/Qowaiv/actions/runs/10856017939/job/30129839393

**Comment by renzo-baasdam** [comment]
Seems to have been resolved by #413.

**Comment by Corniel** [comment]
I think you're right. I was looking at an old issue, not the latest build. Me bad.
